README.md

```diff
@@ -46,6 +46,7 @@ and replace it with the proper embed-code.
   - [Spotify](#spotify)
   - [Twitter](#twitter)
   - [YouTube](#youtube)
+  - [Custom Transformers](#custom-transformers)
```

Comment:
Don't forget to bring this one level higher too 🙂

Reply:
By the way, I ran `npx doctoc README.md` as suggested by the comments and there are some elements in the TOC that are not in the file.

Reply:
Yeah I know, I'd like to not have all sub-headings `usage` & `result` in the TOC, 'cause it's always the same for each built in transformer